php-ts-mode.el

```diff
@@ -276,5 +276,9 @@ Return nil if there is no name or if NODE is not a defun node."
 
   (treesit-major-mode-setup))
 
+(when (treesit-ready-p 'php)
+  (add-to-list 'auto-mode-alist '("/\\.php_cs\\(?:\\.dist\\)?\\'" . php-ts-mode))
+  (add-to-list 'auto-mode-alist '("\\.\\(?:php\\.inc\\|stub\\)\\'" . php-ts-mode)))
```

Comment:
The [`.stub` extension seems to be used by languages other than PHP](https://github.com/search?q=path%3A*.stub&type=code), so we probably shouldn't add it by default.

```suggestion
  (add-to-list 'auto-mode-alist '("\\.\\(?:php[s345]?\\|phtml\\)\\'" . php-ts-mode) t))
```

I think it's fine to do this, but what do you think?

To prioritize `php-ts-mode` over `php-mode` from `php-mode-maybe` set as follows:

```el
(custom-set-variables
 '(php-default-major-mode 'php-ts-mode))
```

Reply:
To be honest for start I think maybe we should just go with php[s345]? only as mixed type with html maybe tricker.